CMakeLists.txt

```diff
@@ -121,13 +111,67 @@ if(EMSCRIPTEN)
     set(EMSCRIPTEN_FEATURES "${EMSCRIPTEN_FEATURES} -s \"EXTRA_EXPORTED_RUNTIME_METHODS=[ENV']\"")
 endif()
 
-find_package(Clang REQUIRED)
-include(AddLLVM)
-include(HandleLLVMOptions)
-add_definitions(${LLVM_DEFINITIONS})
+find_package(CppInterOp REQUIRED CONFIG PATHS "${CMAKE_PREFIX_PATH}" "${CMAKE_PREFIX_PATH}/lib" "${CPPINTEROP_DIR}" "${CPPINTEROP_DIR}/lib")
+if(CppInterOp_FOUND)
+  set(CppInterOp_CLANG_VERSION "17")
```

Comment:
Shouldn't this go in the `CppInterOpConfig.cmake`?

Reply:
See https://github.com/compiler-research/CppInterOp/pull/214. Once we release CppInterOp we will use that instead of the `CppInterOp_CLANG_VERSION`.